README.md

```diff
@@ -21,106 +21,135 @@
+| `component.reader`      | Yes |  Fully qualified name of reader component, e.g.`za.co.absa.hyperdrive.ingestor.implementation.reader.kafka.KafkaStreamReader` |
+| `component.manager`     | Yes |  Fully qualified name of manager component, e.g. `za.co.absa.hyperdrive.ingestor.implementation.manager.checkpoint.CheckpointOffsetManager` |
+| `component.decoder`     | Yes |  Fully qualified name of decoder component, e.g. `za.co.absa.hyperdrive.ingestor.implementation.decoder.avro.confluent.ConfluentAvroKafkaStreamDecoder` |
+| `component.transformer` | Yes |  Fully qualified name of transformer component, e.g. `za.co.absa.hyperdrive.ingestor.implementation.transformer.column.selection.ColumnSelectorStreamTransformer` |
+| `component.writer`      | Yes |  Fully qualified name of writer component, e.g. `za.co.absa.hyperdrive.ingestor.implementation.writer.parquet.ParquetPartitioningStreamWriter` |
+
+##### Spark settings
+| Property Name | Required | Description |
+| :--- | :---: | :--- |
+| `ingestor.spark.app.name` | Yes | User-defined name of the Spark application. See Spark property `spark.app.name` |
+
+#### Settings for built-in components
+##### KafkaStreamReader
+| Property Name | Required | Description |
+| :--- | :---: | :--- |
+| `reader.kafka.topic` | Yes | The name of the kafka topic to ingest data from. Equivalent to Spark property `subscribe` |
+| `reader.kafka.brokers` | Yes | List of kafka broker URLs . Equivalent to Spark property `kafka.bootstrap.servers` |
+
+Any additional properties for kafka can be added with the prefix `reader.option.`. E.g. the property `kafka.security.protocol` can be added as `reader.option.kafka.security.protocol`
+
+See e.g. the [Structured Streaming + Kafka Integration Guide](https://spark.apache.org/docs/latest/structured-streaming-kafka-integration.html) for optional kafka properties.
+
+##### CheckpointOffsetManager
+**Caution**: Currently, the `CheckpointOffsetManager` requires the property `reader.kafka.topic` to be set.
+
+| Property Name | Required | Description |
+| :--- | :---: | :--- |
+| `manager.checkpoint.base.location` | Yes | Used for Spark property `checkpointLocation`. The checkpoint location is resolved to `{manager.checkpoint.base.location}/{topic}`, where `topic` is `reader.kafka.topic` |
+
+##### ConfluentAvroKafkaStreamDecoder
+The `ConfluentAvroKafkaStreamDecoder` is built on [ABRiS](https://github.com/AbsaOSS/ABRiS). More details about the configuration properties can be found there.
+
+| Property Name | Required | Description |
+| :--- | :---: | :--- |
+| `decoder.avro.schema.registry.url` | Yes | URL of Schema Registry, e.g. http://localhost:8081. Equivalent to ABRiS property `SchemaManager.PARAM_SCHEMA_REGISTRY_URL` |
+| `decoder.avro.value.schema.id=latest` | Yes | The schema id. Use `latest` or explicitly provide a number. Equivalent to ABRiS property `SchemaManager.PARAM_VALUE_SCHEMA_ID` |
```

Comment:
`decoder.avro.value.schema.id=latest` this should be only property name without value.

Reply:
Oh yes. Thanks for spotting this one.